components/Item/UpNext.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <v-fade-transition>
+    <v-card class="container pa-4">
+      <div class="d-flex flex-column flex-grow-1">
+        <v-card-title class="countdown-header pa-0">
+          {{ $t('dialog.upNext.nextEpisodePlayingIn') }}
+          <span class="primary--text darken-2">
+            &ensp;{{ timeLeft }} {{ $t('seconds') }}
+          </span>
+        </v-card-title>
+        <v-card-subtitle class="title subtitle-1 pa-0">
+          {{ nextSeriesName }} -
+          {{
+            $t('tvShowAbbrev', {
+              seasonNumber: nextSeasonNumber,
+              episodeNumber: nextEpisodeNumber
+            })
+          }}
+          - <span v-if="$vuetify.breakpoint.xsOnly"> <br /> </span>
```

Comment:
This seems a bit messy. For one, you could probably move the v-if to the `<br />` tag, but also with this line like this, the mobile view would have a dangling `-` at the end of the line.

Reply:
Got rid of the dangling `-` for the mobile view, however it still looks just as messy: ` <span v-if="$vuetify.breakpoint.smAndUp"> - </span> <br v-else />`

Do you have a suggestion for a cleaner way to write it?